doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -3011,3 +3011,47 @@ Example of DHCP log entry (extended logging enabled):
     "client_id":"54:ee:75:51:e0:66",
     "dns_servers":["192.168.1.50","192.168.1.49"]
   }
+
+Event type: ARP
+-----------------
```

Comment:
@jufajardini should there be some less dashes ? To match the length of `Event type: ARP` ?

Reply:
Indeed!